frontend/src/pages/modelRegistry/screens/ModelRegistry.tsx

```diff
@@ -21,9 +19,7 @@ const ModelRegistry: React.FC = () => {
           <EmptyRegisteredModels preferredModelRegistry={preferredModelRegistry?.metadata.name} />
         </>
       }
-      title={
-        <TitleWithIcon title="Registered models" objectType={ProjectObjectType.registeredModels} />
-      }
+      title="Registered models"
```

Comment:
Yes, let's stick with the old design and we can handle that in the redesign part